### DISCUSSION TITLE
Can NativeControlHost be used on X11?

### DISCUSSION THREAD

**Discussion by Coloryr** [AvaloniaUI/Avalonia#14428] [Q&A]
I want to stack two windows together and place them in one window to achieve in-game coverage. I have completed it under Windows, can I do it under Linux

**Comment by thevortexcloud** [comment]
From memory it works. But I have not messed around with it too much. I know VLC Sharp uses it for it's Avalonia integration and it does work.

https://github.com/videolan/libvlcsharp/blob/3.x/src/LibVLCSharp.Avalonia/VideoView.cs

**Reply by Coloryr** [reply]
Yes, this is possible, but my one is a subprocess. Can we also achieve the same effect
